src/templates.ts

```diff
@@ -1031,24 +1032,27 @@ class ComboEditor {
     }
     for (const c in data) {
       const vals = data[c];
-      for (const { shapeCount } of vals) {
+
+      for (let ii = 0; ii < vals.length; ii++) {
```

Comment:
Use "i" as a counter.

As the first line, use
const {shapeCount} = vals[i];

Reply:
Did the second part.

For the variable name, I prefer ii to the standard i, as it makes find/replace for the loop variable much easier.

Reply:
i is how it's written as with the rest of the codebase.  If this is to be the standard, that should be a separate PR that changes every instance of i.

Reply:
For large but incrementally beneficial refactors like this, I prefer to do them piecemeal (i.e. convert to the new standard as code is written, modify old code as it's modified), but I recognize that hurts consistency.